week1/homework/src/server.js

```diff
@@ -1,17 +1,36 @@
-'use strict';
+"use strict";
```

Comment:
Why did all of the single quotes got replaced by double quotes? 😛 It may seem like a non-important detail, but you need to respect the style of the code that you are contributing to. When you will work within a team, one style will be enforced and you need to comply with it, otherwise it's a mess!

Reply:
Thanks for the feedback Alex, 
This is happening because of the ESlint settings, I'll search how to change it to single quotations